components/forms/Form/Form.tsx

```diff
@@ -19,18 +19,32 @@ const InnerForm = (props: InnerFormProps & FormikProps<FormValues>) => {
   const { children, handleSubmit, t } = props;
 
   const errorList = props.errors ? getErrorList(props) : null;
+  const errorId = "gc-form-errors";
   const formStatus = props.status === "Error" ? t("server-error") : null;
   if (formStatus) {
-    window.scrollTo({
-      top: 0,
-      behavior: "smooth",
-    });
+    window.scrollTo({ top: 0, behavior: "smooth" });
```

Comment:
It's a minor issue and doesn't affect functionality, but the "smooth" scroll doesn't appear to be working anymore.

Reply:
it's probably being written by the useEffect below - in which case we can probably just remove this?

Reply:
Good catch, this one pertains to server-side errors from the submission lambda (either a notify error, or connection etc). But, even though it's not part of the validation, you're completely right that it should have the same accessibility fix applied to it, within the useEffect. 

Updated the branch, thanks for your help!